specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```diff
@@ -433,7 +433,12 @@
         "cacheSize": {
           "type": "integer",
           "format": "int32",
-          "description": "The cache size"
+          "description": "The cache size",
+          "x-ms-mutability": [
+            "create",
+            "read",
+            "update"
+          ]
```

Comment:
Please revert this change as it makes no semantic difference -- see the [Autorest docs](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-mutability):

> applying the extension as "x-ms-mutability": ["create", "read", "update"] or not applying it will have the same effect.